video/out/drm_common.c

```diff
@@ -497,19 +499,79 @@ static bool setup_mode(struct kms *kms, const char *mode_spec)
     return false;
 }
 
-static int open_card(int card_no)
+static int open_card_path(const char *path)
 {
-    char card_path[128];
-    snprintf(card_path, sizeof(card_path), DRM_DEV_NAME, DRM_DIR_NAME, card_no);
-    return open(card_path, O_RDWR | O_CLOEXEC);
+    return open(path, O_RDWR | O_CLOEXEC);
+}
+
+static char *get_primary_device_path(struct mp_log *log, int *card_no)
+{
+    drmDevice *devices[DRM_MAX_MINOR] = { 0 };
+    int card_count = drmGetDevices(devices,
```

Comment:
Please use  the `drmGetDevices2` API. The `drmGetDevices` is deprecated, since it always fetches the device PCI revision, causing the device to wake up (from kernel dynamic pm) on older kernels.

Reply:
Would be nice to document that in libdrm!